core/lib/zksync_core/src/lib.rs

```diff
@@ -649,54 +648,138 @@ pub async fn initialize_components(
             .default_priority_fee_per_gas;
         let l1_chain_id = genesis_config.l1_chain_id;
 
-        let eth_client = PKSigningClient::new_raw(
-            operator_private_key.clone(),
-            diamond_proxy_addr,
-            default_priority_fee_per_gas,
-            l1_chain_id,
-            query_client.clone(),
+        let gkms_op_key_name = std::env::var("GOOGLE_KMS_OP_KEY_NAME").ok();
```

Comment:
maybe add

Reply:
In the PR, if we had setup the GOOGLE_KMS_OP_KEY_NAME variable, the zk server will choose GKMSSigningClient, otherwise will go PKSigningClient. I think the logic in v25 will be different, maybe there are other signers.